Makefile

```diff
@@ -92,8 +96,13 @@ install:
 	install -m 644 $(wildcard docs/*.8.md) $(PREFIX)/share/man/man8
 	install -d $(PREFIX)/share/man/man5
 	install -m 644 $(wildcard docs/*.5.md) $(PREFIX)/share/man/man5
+	install -D -m 644 ocid.service $(PREFIX)/lib/systemd/system
+	install -D -m 644 ocid.conf $(DESTDIR)/etc
 
 uninstall:
+	systemctl stop ocid.service
+	systemctl disable ocid.service
```

Comment:
I don't think we should've included this in `uninstall`. If you're inside a package manager, it should be handled by the package.